x-pack/test/apm_api_integration/tests/alerts/chart_preview.spec.ts

```diff
@@ -219,8 +225,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
       expect(response.status).to.be(200);
       expect(response.body.latencyChartPreview[0].data[0]).to.eql({
-        x: 1627974600000,
-        y: 18485.85714285714,
+        x: 1627973700000,
+        y: 2399148.714285714,
```

Comment:
Right now these values are not very useful. It's not possible for me as a reviewer to verify whether the numbers are correct. I think this is partly due to the fact that this test uses archives, instead of synthtrace. If you can change that, that would be awesome!

Reply:
Updated tests to use Synthtrace.